unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/WebRequest/WebRequestController.cs

```diff
@@ -160,13 +160,13 @@ public static WebRequestController Create()
                         }
                         else
                         {
-                            OnFail?.Invoke(resultOp.webRequest);
+                            OnFail?.Invoke(resultOp);
```

Comment:
Same here.

Reply:
This is a necessary step in order to move the WebRequestAsync operation to an interface, since the webrequest is a class from unity it can't derive from an interface